inc/consent/namespace.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Altis\Core\Consent;
+
+use Altis;
+use Altis\Consent\Settings;
+
+/**
+ * Kick it off.
+ */
+function bootstrap() {
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\set_consent_options' );
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugins', 1 );
+
+	add_filter( 'altis.analytics.noop', __NAMESPACE__ . '\\set_analytics_noop' );
+	add_filter( 'altis.analytics.data', __NAMESPACE__ . '\\set_analytics_data', 1 );
+
+	// If statistics hasn't been consented to, don't load the GTM output.
+	if ( ! wp_get_consent( 'statistics' ) ) {
```

Comment:
`wp_get_consent()` doesn't appear to be a function 😛 
```suggestion
	if ( ! wp_get_consent_type( 'statistics' ) ) {
```